src/cron/workflow.ts

```diff
@@ -0,0 +1,36 @@
+import { Context } from "../types";
+import db from "./database-handler";
+
+export async function updateCronState(context: Context) {
+  await db.update((data) => {
+    for (const key of Object.keys(data)) {
+      if (!data[key].length) {
+        delete data[key];
```

Comment:
A much simpler implementation probably might be to only store the last run time in posix time and just clobber the value. For example, the "database" can be a file called .lastrun
```
1718692310
```

We might even be able to store this elsewhere like in the secrets or repository description to avoid polluting the color commit history.